README.md

```diff
@@ -158,9 +166,599 @@ search.start();
 - If you use Vue, check out [meilisearch-vue](https://github.com/meilisearch/meilisearch-vue/)
 - If you use Angular, check out [meilisearch-angular](https://github.com/meilisearch/meilisearch-angular/)
 
-## Compatibility with MeiliSearch
+## API resources
+
+List of all the components that are available in [instantSearch](https://github.com/algolia/instantsearch.js) and their compatibilty with [MeiliSearch](https://github.com/meilisearch/meilisearch-react/).
+
+### ✅ InstantSearch
+
+[instantSearch references](https://www.algolia.com/doc/api-reference/widgets/instantsearch/js/)
+
+`instantSearch` is the main component. It manages the widget and lets you add new ones.
+
+- ✅ IndexName: Name of your index. _required_
+- ✅ SearchClient: Search client, in our case instantMeiliSearch. See [customization](#customization) for details on options. _required_
+- ❌ numberLocale: Did not work with both algoliasearch and instantMeiliSearch.
+- ✅ searchFunction: Surcharge the search function provided by the search client.
+- ✅ initialUiState: Determine the search state on app start.
+- ✅ onStateChange: Change search state on change (see option above).
+- ✅ stalledSearchDelay: Time in ms before search is considered stalled. [Used for loader](https://www.algolia.com/doc/guides/building-search-ui/going-further/improve-performance/js/#using-the-searchbox).
+- ✅ routing: browser URL synchronization, search parameters appear in current URL ([guide](https://www.algolia.com/doc/guides/building-search-ui/going-further/routing-urls/js/)).
+- ✅ insightsClient: Hook analytics to search actions ([see insight section](#insight)).
+
+```js
+const search = instantsearch({
+  indexName: 'instant_search',
+  searchClient: instantMeiliSearch(
+    'https://demos.meilisearch.com',
+    'dc3fedaf922de8937fdea01f0a7d59557f1fd31832cb8440ce94231cfdde7f25',
+    {...instantMeiliSearchOptions}
+  ),
+  {...instantSearchOptions}
```

Comment:
Why `{}` around? It looks like `instantsearch` is defined like this:
```js
const search = instantsearch({
  indexName: '...',
  searchClient: ...,
  option1: ...,
  option2: ...,
  ...
});
```

and not:

```js
const search = instantsearch({
  indexName: '...',
  searchClient: ...,
  {
    option1: ...,
    option2: ...,
    ...
  }
});
```